scripts/helper_numpy.py

```diff
@@ -0,0 +1,83 @@
+#!/usr/bin/env python
+# coding: utf8
+'''Compute a Numpy friendly data-structures from VGGish files
+Example
+-------
+$ cd {repo_root}
+$ python ./scripts/helper_numpy.py --csv_file /path/to/sparse-labels.csv \
+--vggish_path /path/to/vggish/ --output_file /path/to/output.npz
+
+Any valid output file '*.npz' has to be speficied by the user
+'''
+
+from __future__ import print_function
+import argparse
+import numpy as np
+import os
+import pandas as pd
+import sys
+import json
+
+
+def main(csvfile, vggishpath, outfile):
+
+    success = []
+    df = pd.read_csv(csvfile)
+    instruments = np.unique(df['instrument'])
+    sample_key = np.unique(df['sample_key'])
+    songs_num = len(sample_key)
+    inst_num = len(instruments)
+
+    print('Extracting the vggish features...')
+    X = np.empty([songs_num, 10, 128], dtype=int)
+    count = 0
+    for sk in sample_key:
+        sk_prefix = sk[:3]
+        full_name = os.path.join(vggishpath, sk_prefix, sk + '.json')
+        with open(full_name, 'r') as f:
+            X_tmp = json.load(f)
+        X[count] = X_tmp['features']
+        count += 1
+
+    print('Extracting the labels information...')
+    Y_true = 0.5 * np.ones([songs_num, inst_num], dtype=float)
+    Y_mask = np.zeros([songs_num, inst_num], dtype=bool)
+
+    inst_lab = np.copy(df)
+    for inst in inst_lab:
+        x_pos = int(np.arange(songs_num)[sample_key == inst[0]])
+        y_pos = int(np.arange(inst_num)[instruments == inst[1]])
+        Y_true[x_pos, y_pos] = inst[2]
+        Y_mask[x_pos, y_pos] = 1
```

Comment:
in this implementation, this could (eventually) be `Y_mask[x_pos, y_pos] = inst[3]`

Reply:
Could you point me to the new label csv file so I can use the appropriate name/format?

Reply:
done

Reply:
@bmcfee would you vote for a `Y_mask` with the number of responses for a class (3, 0, 30, ...) or a binary `Y_mask` with the presence/absence of a response for a class (True, False, True, ...).

Reply:
I'd vote for binary mask indicating whether the (track, tag) pair has been measured.  More fine-grained information can be coded elsewhere.